# Status of optimization
std_msgs/Header header

float64 current_objective
float64 secs_since_start